raiden-ts/src/config.ts

```diff
@@ -65,7 +68,8 @@ export const RaidenConfig = t.readonly(
       expiryFactor: t.number, // must be > 1.0
       httpTimeout: t.number,
       discoveryRoom: t.union([t.string, t.null]),
-      pfs: t.union([t.readonlyArray(t.union([Address, t.string])), t.boolean]),
+      additionalServices: t.readonlyArray(t.union([Address, t.string])),
+      pfsMode: PfsModeC,
```

Comment:
Nitpick & voluntary: `pfsSelectMode` Being more precise now might help us later if we need to extend the configuration for reasons we can't know yet. (ofc this goes also for the type name if you decide to do so)